docs/UserGuide.md

```diff
@@ -1,28 +1,65 @@
 
 # VendorVault User Guide
 
-VendorVault is a **desktop app for managing supplier contact information and deliveries, optimized for use via a Command Line Interface** (CLI). If you can type fast, VendorVault can get your contact management tasks done faster than traditional GUI apps. VendorVault specifically targets small convenience/grocery stores and aims to help them achieve higher efficiency! 
+VendorVault is a **desktop app for managing supplier contact information and deliveries, optimized for use via a Command Line Interface** (CLI). If you can type fast, VendorVault can get your contact management tasks done faster than traditional GUI apps. VendorVault specifically targets small convenience/grocery stores and aims to help them achieve higher efficiency!
 
 --- 
-
-### Table of Contents will be added here by Linkes
-
-
----
+# Table of Contents
+<!-- * Table of Contents -->
+1. [How to get started]()
+2. [Setting up VendorVault ]()
```

Comment:
Not sure if the extra space before the bracket can affect formating

Reply:
No I dont think it will affect formatting i will change it